docs/standalone_build_guide.md

```
@@ -0,0 +1,42 @@
+
+## Standalone Build Guide
+
+The standalone build (`npm run standalone`) exposes the `mdv` object (including `ChartManager` and `DataLoaders`) on the `window`. It is up to you, the developer, to create dataloaders and listeners as needed.
+
+### How to Use
+
+1. **Build**: Run the standalone build script:
+    ```
```

markdownlint-cli2 (0.20.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)